content/en/blog/_posts/2022-12-12-host-process-containers-stable/index.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Windows HostProcess Containers Are Going Stable"
+date: 2022-12-13
+slug: host-process-containers-stable
```

Comment:
```suggestion
slug: windows-host-process-containers-ga
```
(`windows-` is for improved visibility to search engines)

Reply:
updated - thanks for the tip!